pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+POP: 1
+Title: POP Purpose and Guidelines
+Version: $Revision$
+Last-Modified: $Date$
+Author: Carol Willing
+Status: Active
+Type: Process
+Content-Type: text/x-rst
+Created: 14-Apr-2015
+Post-History: 14-Apr-2015
+
+
+What is a POP?
+==============
+
+POP stands for PyLadies Organizer Proposal.  A POP is a design
+document providing information to the PyLadies community, or describing
+a new feature for PyLadies or its processes or environment.  The POP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend POPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into PyLadies.  The POP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the POPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+proposal [1]_.
+
+
+POP Types
+=========
+
+There are three kinds of POP:
+
+1. A **Standards Track** POP describes a new feature or implementation
```

Comment:
For PyLadies, I think `a new feature or implementation` isn't really relevant to us.  Perhaps the standards track is actually "principles" and/or "policies".